docs/source/develop.rst

```diff
@@ -166,6 +166,11 @@ You can also test certain modules or individual tests for faster response::
 
    py.test dask/dataframe/tests/test_dataframe.py::test_rename_index
 
+If you want the tests to run faster, you can run them in parallel using
+``pytest-xdist``::
+
+   py.test dask --verbose -n auto
```

Comment:
I realize this predates this PR, but since we specify `-v` as one of our default `pytest` options

https://github.com/dask/dask/blob/e32d1adcd9231b95e6b68b37a47ebaf753ea4efb/setup.cfg#L45

the explicit `--verbose` flag here isn't needed as it will automatically be applied. Would you mind removing it here and other places it occurs?

Reply:
ooh good call :)